tests/dataset_fixture.py

```diff
@@ -44,7 +47,7 @@ def update_spreadsheet_project_shortname(self, new_shortname):
 
     @property
     def metadata_spreadsheet_path(self, metadata_version="5"):
-        xlsx_files = glob.glob(f"{self.dataset_path}/*_integration_test_*.xlsx")
+        xlsx_files = glob.glob(f"{self.dataset_path}/*.xlsx")
```

Comment:
Globbing seems a little unnecessary now that the spreadsheet filename is in the README.json.
Isn't this just `return os.path.join(self.dataset_path, os.path.basename(self.config['spredsheet_location']))`